xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
this strangely didn't produce any errors, the only test that is failing is the one that i added to check if XONSH_INTERACTIVE is true if the `-i` is used along with a `filename`. Xonsh atm doesn't seem to be able to start in interactive mode when a file is passed @scopatz